app/controllers/filter_controller.rb

```diff
@@ -1,7 +1,7 @@
 class FilterController < ApplicationController
   def index
     # REFACTOR: Make this set of attributes have a single source of accessible truth
-    filtered_params = params.permit([:category, :hero, :map, :from, :to, :sort, :expired, :author, :players, :language, :search, :default, :page])
```

Comment:
What was `:default` even?

Reply:
Default locale I believe. Either way, it gets added with the redirected request again, so it is not necessary to include.